README.md

```diff
@@ -137,6 +137,10 @@ swagger_path :index do
 end
 ```
 
+The `swagger_path` macro layer is just some syntactic sugar over regular elixir functions. Therefore it can easily be extended, for instance, if we want to reuse some common parameters.
+
+For more details on this take a look at [Reusing Swagger Parameters](https://hexdocs.pm/phoenix_swagger/reusing-swagger-parameters.html).
```

Comment:
Looks like you can just use a relative link: 

```markdown
[Reusing Swagger Parameters](reusing-swagger-parameters.html)
```

I noticed this is how the phoenix guides link between each other:

```markdown
For now, we'll ignore the pipelines and the use of `scope` here and just focus on adding a route. (We cover these topics in the [Routing Guide](routing.html), if you're curious.)
```